Tests/RxSwiftTests/Observable+CombineLatestTests+arity.tt

```diff
@@ -119,7 +119,7 @@ extension ObservableCombineLatestTest {
         let factories: [(<%= (Array(0..<i).map { _ in "TestableObservable<Int>" }).joined(separator: ", ") %>) -> Observable<Int>] =
```

Comment:
This is the source of truth for `Tests/RxSwiftTests/Observable+CombineLatestTests+arity.swift`. This should probably be fixed also.